### ISSUE TITLE
HSLA valid range discussion

### ISSUE THREAD

**Issue by JMS55** [bevyengine/bevy#7095]
[bevy::prelude::Color::Hsla::lightness](https://docs.rs/bevy/latest/bevy/prelude/enum.Color.html#variant.Hsla.field.lightness)

The docs say that the valid range is [0, 1]. However, when it comes to HDR rendering, lightness > 1.0 can be used to make a color more bright, and is useful for instance with bloom. 

Should we change the docs? Introduce a separate "brightness" field to sprites or something? What's the right solution?

**Comment by alice-i-cecile** [comment]
IMO we should change the docs here to explain better.

**Comment by JMS55** [comment]
Agreed. Also after experimenting with this myself, it's not HSLA specifically, but color values > 1.0 in general.

**Comment by CptPotato** [comment]
I'm pretty sure HSL is not valid (or useful) beyond a lightness of `1.0` because it fades to white at `1.0` and it's unclear how larger values should behave. HSV *could* work since `Value` is just a multiplier, so it can be used extend the brightness range.

Either way, `Color::Hsla`, `Color::Rgba` (and potentially `Color::Hsva`) all specify colors in the SRGB space which includes gamma correction and is limited to `[0.0, 1.0]` from what I understand.

`RgbaLinear` on the other hand has no such limitations, so I think it's a good idea to adjust the docs of this variant, possibly also clarifying that a value of `(1.0, 1.0, 1.0, 1.0)` is fully white.

**Comment by alice-i-cecile** [comment]
These docs still need improvement even with the new `bevy_color`.
